package.json

```diff
@@ -64,6 +64,7 @@
     "filetype": "./content/src/filetype.js",
     "gadget": "./content/src/gadget.js",
     "guide": "./content/src/guide.js",
+    "html2canvas": "./content/lib/html2canvas.js",
```

Comment:
It is possible but it doesn't work yet :stuck_out_tongue: I have filed an issue for them.